app/models/category.rb

```diff
@@ -0,0 +1,7 @@
+class Category < Classification
+  default_scope { where(:parent_id => 0) }
```

Comment:
@gtanzillo in particular, this feels a bit strange to me. And the `default_scope` isn't quite right - it should be leveraging `Classification#categories`, but unfortunately that doesn't return a relation.

On the plus side, creating this object lets the API code Just Work for the most part. Any thoughts?